src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -754,14 +760,48 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
         if (ptr.PointerDeviceType() == Windows::Devices::Input::PointerDeviceType::Mouse)
         {
-            if (point.Properties().IsLeftButtonPressed())
+            if (_terminal->IsSelectionActive() && point.Properties().IsLeftButtonPressed())
             {
                 const auto cursorPosition = point.Position();
-                const auto terminalPosition = _GetTerminalPosition(cursorPosition);
+                _SetEndSelectionPointAtCurstor(cursorPosition);
 
-                // save location (for rendering) + render
-                _terminal->SetEndSelectionPosition(terminalPosition);
-                _renderer->TriggerSelection();
+                const double cursorBelowBottomDist = cursorPosition.Y - _root.Padding().Top - _swapChainPanel.ActualHeight();
+                const double cursorAboveTopDist = -cursorPosition.Y + _root.Padding().Top;

```

Comment:
That little `-` makes me uncomfortable. So easy to miss. Just swap the way you mention the operands and make it a subtraction instead of an addition.

Reply:
Or write something like `(-1.0)*` instead to make it less sneaky. That way you could keep the similarity to the line above it.